policyengine_us/variables/gov/states/hi/tax/income/credits/cdcc/dependent_care_benefit/hi_cdcc_earned_income.py

```diff
@@ -0,0 +1,43 @@
+from policyengine_us.model_api import *
+
+
+class hi_cdcc_earned_income(Variable):
+    value_type = float
+    entity = Person
+    label = "Hawaii CDCC earned income"
+    defined_for = StateCode.HI
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=28"
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=29"
+        "https://files.hawaii.gov/tax/legal/hrs/hrs_235.pdf#page=41"
+        "https://files.hawaii.gov/tax/forms/2022/schx_i.pdf#page=2"
+    )
+
+    def formula(person, period, parameters):
+        # Head or spouse are eligible for an income floor if disabled or a student
+        income_floor_eligible = person("is_disabled", period) | person(
+            "is_full_time_student", period
+        )
+        income = person("earned_income", period)
+        increased_income = person("hi_cdcc_eligible_income_floor", period)
+        uncapped_income = where(
+            income_floor_eligible,
+            increased_income,
+            income,
+        )
+        # Edge case: both spouses were students or disabled:
+        # If both filers are disabled / student below the floor limit,
+        # only one person with larger earning gets elevated to the floor
+        # If both filers are disabled / student but only one person is below the floor limit,
+        # then the person below the floor limit will be elevated to the floor
+        below_income_floor = income_floor_eligible & (income < uncapped_income)
+        more_than_one_below_floor = sum(below_income_floor) > 1
```

Comment:
test this as well. though i wonder if it can go in the tax unit computation so this remains fully person-level
```suggestion
        more_than_one_below_floor = tax_unit.sum(below_income_floor) > 1
```

Reply:
Hi Max, I tested your suggestion. However, tax_unit cannot be used in person-level formula and the further calculation requires other person-level variables in this formula so I cannot move this line to its parent formula, which is a tax_unit level formula. I use `person.tax_unit.sum` in the end. I hope it also works.

Reply:
@MaxGhenis I think if we move the the computation to the `hi_cdcc_min_head_spouse_earned` (the tax_unit variable) then we would to return smaller_income and uncapped_income for the person level variable which we don't think is possible. Thus we capped it here